### PR TITLE
Add Uniaxial compression into docs

### LINKED ISSUE #599: Add uniaxial compression into docs

### PR DESCRIPTION
**Pull request by mvanzulli**
Closes #599 
Deploy: [docs](https://onsas.github.io/ONSAS.m/previews/PR600/examples/uniaxialCompression/)

Just a detail: PR number = 600  💯  🚀 🤓

### CODE REVIEW

**Comment by mvanzulli** [comment]
I have solved the problem considering different neo-Hookean models but I couldn't find the one who gives me the cosserat stress tensor implemented in ONSAS. I have tried with:

```math
 \Psi(\mathbf{C}) = \frac{\mu}{2} (I_1 - 3) - \mu \ln(J) + \frac{\lambda}{2} \ln(J)^2,
``

and 

 ```math
 \Psi(\mathbf{C}) = \frac{\mu}{2} (I_1 - 3 - 2\ln(J))
 ``

**Comment by jorgepz** [comment]
Excellent! You found the strain energy function. If is the isotropic term of the function shown in Equation 2 in [this paper](https://www.sciencedirect.com/science/article/abs/pii/S0957417415003942?via%3Dihub).

To obtain the cosserat derivative it could be useful to see that
```math
det(C) = J^2
```

File: examples/uniaxialCompression/uniaxialCompression.m [hunk 1]
```diff
@@ -1,6 +1,53 @@
 %md# Uniaxial Compression example
+%md
+%md[![Octave script](https://img.shields.io/badge/script-url-blue)](https://github.com/ONSAS/ONSAS.m/blob/master/examples/uniaxialCompression/uniaxialCompression.m)
+%md
+%mdIn this tutorial example an hyper-elastic solid is submitted to a uniaxial compression test. The geometry and tension applied are shown in the figure, where the $Lx$, $Ly$ and $Lz$ are the dimensions and the compression $p$ is applied on the face $x=Lx$, as nominal traction.
```
Comment (jorgepz) [inline examples/uniaxialCompression/uniaxialCompression.m:5]:
```suggestion
%mdIn this tutorial example an hyper-elastic solid is submitted to a uniaxial compression test. The geometry and tension applied are shown in the figure, where the $Lx$, $Ly$ and $Lz$ are the dimensions, and the compression $p$ is applied on the face $x=Lx$, as nominal tension.
```

File: examples/uniaxialCompression/uniaxialCompression.m [hunk 2]
```diff
@@ -1,6 +1,53 @@
...
+%md\textbf{F} = \left[ \begin{matrix} \alpha & 0 & 0 \\ 0 & \beta & 0 \\ 0 & 0 & \beta \end{matrix} \right]
+%md\qquad
+%md\textbf{E} = \frac{1}{2}(\textbf{C} - \textbf{I}) = \left[  \begin{matrix} \frac{1}{2} \left(\alpha^2 -1\right) & 0 & 0 \\ 0 &  \frac{1}{2} \left(\beta^2 -1\right) & 0 \\ 0 & 0 &  \frac{1}{2} \left(\beta^2 -1\right) \end{matrix}
+%md\right]
+%md```
+%mdThe neo-Hookean elastic strain energy potential $\Psi$ is given by:
+%md```math
+%md\Psi(\mathbf{C=\textbf{F}^T\textbf{F}}) = \frac{\mu}{2}(I_1 -3) + K ( J^2 -1 )
+%md```math
+%mdwhere $I_1 = \mathrm{tr}(\mathbf{C})$ is the first invariant, $J = \sqrt{\det(\mathbf{C})}$ and $K$ and $\mu$ material parameters.
```
Comment (jorgepz) [inline examples/uniaxialCompression/uniaxialCompression.m:27]:
```suggestion
%mdwhere $I_1 = \mathrm{tr}(\mathbf{C})$ is the first invariant, $J = \sqrt{\det(\mathbf{C})}$ and $K$ and $\mu$ are the bulk and shear material parameters, respectively.
```

File: examples/uniaxialCompression/uniaxialCompression.m [hunk 3]
```diff
@@ -1,6 +1,53 @@
...
+%md```math
+%md\textbf{S}( \mathbf{C} ) = \mu (\textbf{I} - \mathbf{C}^{-1}) + K (J(J-1)\mathbf{C}^{-1}) 
+%md```
+%mdthen, using the relation $\textbf{P}=\textbf{F}\textbf{S}$, the $P_{yy}=P_{zz}$ component is computed: and set to zero (using the boundary conditions)
+%md```math
+%mdP_{yy}( \mu,K ) =
+%md2\beta \left( 
+%md    \frac{\mu}{2} -  \frac{\mu}{\beta^2} + K (\alpha^2\beta^2 -\alpha) 
+%md \right) = 0
+%md```math
```
Comment (jorgepz) [inline examples/uniaxialCompression/uniaxialCompression.m:39]:
```suggestion
%md```
```

File: examples/uniaxialCompression/uniaxialCompression.m [hunk 4]
```diff
@@ -101,8 +147,11 @@
```
Comment (jorgepz) [inline examples/uniaxialCompression/uniaxialCompression.m:147]:
```suggestion
```

File: examples/uniaxialCompression/uniaxialCompression.m [hunk 5]
```diff
@@ -133,5 +187,5 @@
 set(labx, 'FontSize', plotfontsize); set(laby, 'FontSize', plotfontsize) ;
 %md
 %md```@raw html
```
Comment (jorgepz) [inline examples/uniaxialCompression/uniaxialCompression.m:189]: I suggest including the plots before merging. You can check that in the deploy https://onsas.github.io/ONSAS.m/previews/PR600/examples/uniaxialCompression/

**Review by jorgepz — APPROVED** [review]

File: examples/uniaxialCompression/uniaxialCompression.m (hunk 5 above)
Comment (mvanzulli) [reply examples/uniaxialCompression/uniaxialCompression.m:190]: Done

**Pull request merged**